src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+$columns: 7;
+$cell-size: 100px;
+$gap: 1px;
+$days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5 , sun 6;
+$max-width: $cell-size * $columns + $gap * $columns;
+
 body {
   margin: 0;
 }
+
+.calendar {
```

Comment:
let's split your main file into separate files:
1. in the `styles` folder, create `utils` folder and put 2 files there:
variables.scss and reset.scss with `body { margin: 0; }`